app/api/get-host.js

```diff
@@ -0,0 +1,9 @@
+const { apimConfig } = require('../config')
+
+const getHost = (headers) => {
+  return headers.email ? apimConfig.hostInternal : apimConfig.hostExternal
```

Comment:
Thoughts are hosting the APIs again to resolve this issue so need to work out which host to use depending on if internal user